src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ReplyToMessage.vue

```diff
@@ -116,101 +93,56 @@ export default defineComponent({
   },
   data(): ReplyToMessageData {
     return {
-      selectedFiles: [],
       text: " ",
       Rules,
       showCloseDialog: false,
+      areAttachedFilesValid: true,
+      isFileUploadInProgress: false,
     };
   },
   methods: {
     async handleReplyToMessage() {
-      if (this.text.trim().length > 0) {
-        const { error } = await sendMessage({ communication: { id: this.messageId, text: this.text } });
-        if (error) {
-          this.alertStore.setFailureAlert("Sorry, something went wrong and your changes could not be saved. Try again later.");
+      if (!this.isFileUploadInProgress) {
```

Comment:
Something to consider, using v-form for validation. 

Below reads like a nested if statement. 
if files are in progress show this alert message 
if files are too large show this alert message 
if there is no text show another alert message

We could just run validate on a v-form and gather all the errors from rules in an input component too 

example 

![image](https://github.com/user-attachments/assets/9a29706e-60bc-47dc-95f0-6b0127f65c93)
^^^ the blob at the end is what gets run after running validate on a vForm. Also gives an array of all the errors that are produced.

Reply:
Source: https://vuetifyjs.com/en/components/forms/#exposed-properties 

Example: https://github.com/bcgov/ECC-ECER/blob/0b09bad73b9843a7d6117c32b00bbe5064283f6c/src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceEducation.vue#L320